### PR TITLE
SpeedDial position property

### LINKED ISSUE #1571: SpeedDial does not have a positioning property
**Bug** 
The first thing someone wants to do with the speed dial is position it (usually in the bottom right hand corner). There is currently no option for this

### PR DESCRIPTION
**Pull request by aciccarello**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [x] Unit tests are included in the PR
* [ ] For new widgets, an entry has been added to the `.dojorc`
* [ ] For new widgets, `theme.variant()` is added to the root domnode
* [ ] Any widget variant uses `theme.compose` like [this](https://github.com/dojo/widgets/issues/847)
* [x] WidgetProperties are exported

**Description:**
This adds a property to quickly fixed-position the speed dial either in the bottom right or bottom center. In those cases it will default to a direction of "up".

The floating-action-button previously had no background, but this is problematic when displaying over other content so I removed that style.

### Open questions:
* [ ] Should the direction attribute be allowed at all with positioning?
* [ ] Are these the positions that we want?
* [ ] Does the button style change need to be called out as a breaking change?

Resolves #1571

## Screenshots
localhost_9999_(iPad) (3)
localhost_9999_(iPad) (2)
localhost_9999_(iPad) (6)
localhost_9999_(iPad) (4)
localhost_9999_(iPad) (5)

### CODE REVIEW

File: src/examples/src/config.tsx [hunk 1]
```diff
@@ -1659,6 +1660,11 @@ export const config = {
 					module: DirectionSpeedDial,
 					title: 'With a direction'
 				},
+				{
+					filename: 'Position',
```
Comment (matt-gadd) [inline src/examples/src/config.tsx:1664]: can you make this example use the iframe and sizing so it correctly displays in the example area?

```ts
  sandbox: true,
  size: 'large'
````

**Review by matt-gadd — CHANGES_REQUESTED** [review]
I think the positional change should apply to the floating action button too (given it's the singular version of a speed dial).

**Comment by tomdye** [comment]
@aciccarello do you think this should have a bottom-left option too? Also how about putting the example in a large sandbox so that the action button does not bleed out of the example area?

File: src/examples/src/config.tsx (hunk 1 above)
Comment (tomdye) [reply src/examples/src/config.tsx:1664]: same catch as me - but more helpful 👍

**Comment by aciccarello** [comment]
@matt-gadd I can add this to the FAB as a followup PR

**Review by matt-gadd — APPROVED** [review]

**Comment by aciccarello** [comment]
I've updated styles to use variables for offset position and `translateX`/`translateY` to center the position following some learning from the FAB PR (#1670). Also improved the example so that unsupported directions can't be used with the position.

**Review by matt-gadd — APPROVED** [review]

**Pull request merged**